src/uu/chmod/src/chmod.rs

```diff
@@ -35,14 +35,71 @@ mod options {
     pub const FILE: &str = "FILE";
 }
 
-#[uucore::main]
-pub fn uumain(args: impl uucore::Args) -> UResult<()> {
-    let mut args = args.collect_lossy();
+// "-w -w" is the same as "-w,-w"
+// "-w file -w" is the same as "-w,-w file"
+// "-- -w file" is the same as "-w file"
```

Comment:
Could you expand this a little bit more into a full story? I.e. include why this isn't possible with clap, what edge cases this needs to cover, what it does and how?

Reply:
Tried to clean it up, please check if sufficient :)

Reply:
i wonder if it would be possible to write unit test for this ? :)

Reply:
I agree with Sylvestre and I'd like to see a proper docstring for this function.

Reply:
Sure :) will fix it and unit test later when I am at home